test/suites/security.sh

```diff
@@ -161,6 +161,22 @@ test_security_protection() {
 
   lxc profile unset default security.protection.delete
 
+  # Test start protection
+  lxc init testimage c1
```

Comment:
I dont think we need these 3 steps as we have plenty of container start tests :)

Reply:
Sorry, which 3 steps? Did you mean the 3 steps below?
```
  lxc init testimage c1
  lxc start c1
  lxc delete c1 --force
```

Reply:
yes, this doesnt test anything we dont already test

Reply:
Done.